com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Camera/HDCameraUI.Rendering.Drawers.cs

```diff
@@ -40,10 +44,40 @@ partial class Rendering
                 )
             );
 
+            public static readonly CED.IDrawer Drawer = CED.AdditionalPropertiesFoldoutGroup(CameraUI.Rendering.Styles.header, Expandable.Rendering, k_ExpandedState, AdditionalProperties.Rendering, k_AdditionalPropertiesState, RenderingDrawer, Draw_Rendering_Advanced);
+
+            internal static void RegisterEditor(HDCameraEditor editor)
+            {
+                k_AdditionalPropertiesState.RegisterEditor(editor);
+            }
+
+            internal static void UnregisterEditor(HDCameraEditor editor)
+            {
+                k_AdditionalPropertiesState.UnregisterEditor(editor);
+            }
+
+            [SetAdditionalPropertiesVisibility]
+            internal static void SetAdditionalPropertiesVisibility(bool value)
+            {
+                if (value)
+                    k_AdditionalPropertiesState.ShowAll();
+                else
+                    k_AdditionalPropertiesState.HideAll();
+            }
+
+            static void Draw_Rendering_Advanced(SerializedHDCamera p, Editor owner)
+            {}
+
             static void Drawer_Rendering_AllowDynamicResolution(SerializedHDCamera p, Editor owner)
             {
                 CameraUI.Output.Drawer_Output_AllowDynamicResolution(p, owner);
 
+                var dynamicResSettings = HDRenderPipeline.currentAsset.currentPlatformRenderPipelineSettings.dynamicResolutionSettings;
+                s_IsRunningTAAU = p.allowDynamicResolution.boolValue && dynamicResSettings.upsampleFilter == UnityEngine.Rendering.DynamicResUpscaleFilter.TAAU && dynamicResSettings.enabled;
+
+                if (s_IsRunningTAAU)
+                    EditorGUILayout.HelpBox(Styles.taauInfoBox, MessageType.Info);
```

Comment:
This should probably get cleaned up:
Instead of failing when another TAA algorithm is chosen, turn off DRS / TAAU and instead in the warning say that dynamic resolution is disabled.
My point is that we shouldnt allow the pipeline to get into a corrupted config so easily, even if we add warnings.